circle.yml

```diff
@@ -0,0 +1,9 @@
+machine:
+  node:
+    version: 5.3.0
```

Comment:
It was the last version I used in a major app, in March of this year. I see that 6.0 was released in April, so we can switch to 6.2.0 if necessary. There wasn't a real reason beyond that. Are we supposed to use 4.x.x in our projects?